src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -8506,16 +8490,6 @@ private static string WinInternalGetTarget(SafeFileHandle handle)
                         targetDir = Encoding.Unicode.GetString(reparseMountPointDataBuffer.PathBuffer, reparseMountPointDataBuffer.SubstituteNameOffset, reparseMountPointDataBuffer.SubstituteNameLength);
                         break;
 
-                    case IO_REPARSE_TAG_APPEXECLINK:
```

Comment:
The const string `IO_REPARSE_TAG_APPEXECLINK` is not used anywhere else. Better to remove it.

Reply:
It's still used on line 8289 as a check for OneDrive reparse points

Reply:
Right, I missed that :)